scheduler/src/cook/kubernetes/api.clj

```diff
@@ -275,7 +275,9 @@
           node-name (some-> node .getMetadata .getName)
           pods-on-node (num-pods-on-node node-name pods)
           labels-on-node (or (some-> node .getMetadata .getLabels) {})
-          matching-node-blocklist-labels (some #(get labels-on-node %) node-blocklist-labels)]
+          label-names-on-node (into #{} (keys labels-on-node))
```

Comment:
I don't see where this variable is being consumed.

Reply:
Good catch. My IDE flaked.